estuary.env

```diff
@@ -0,0 +1,8 @@
+set -a
+FULLNODE_API_INFO=wss://api.chain.love
+# FULLNODE_API_INFO=http://10.32.32.1:10000
+
+set +a
+
+
+# EST3edfbe00-2bf0-4c4f-ba5d-81bd0cb5ffd6ARY
```

Comment:
should we add this to `.gitignore`? also guessing this key is for your local estuary but flagging anyways